graphql/data/marathon/deployments/server.js

```diff
@@ -0,0 +1,19 @@
+import DataLoader from 'dataloader';
+
+import config from '../../../config';
+import { fetchWithAuth } from '../../../utils/fetch';
+
+export default class DeploymentsServerConnector {
+  constructor({ authToken }) {
+
+    const getDeployments = fetchWithAuth(authToken, {
+      baseURI: config.endpoints.deployments
+    });
+
+    this.deploymentsLoader = new DataLoader(getDeployments, { batch: false });
+  }
+
+  get() {
+    return this.deploymentsLoader.load('');
```

Comment:
Correct. The value we pass in here also gets appended to the base URL. So we could do `getById` and pass in 'deploymentId` and it will cache that value for the `deploymentId` too.